bouncer/authentication/models.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.utils.crypto import get_random_string
+
+
+class AuthToken(models.Model):
+    key = models.CharField(_("Key"), max_length=32, primary_key=True, editable=False)
+    user = models.ManyToManyField(
```

Comment:
If user is a many-to-many field and a token is assigned to 2 users, which user will be authenticated when the token is used? Maybe you need to change this to a different field.